R/router.R

```diff
@@ -65,16 +65,20 @@ get_url_hash <- function(session = shiny::getDefaultReactiveDomain()) {
 #'
 #' @param path Website route.
 #' @param ui Valid Shiny user interface.
-#' @param server Function that is called as callback on server side
+#' @param server Function that is called as callback on server side [deprecated]
 #' @return A route configuration.
 #' @examples
 #' \dontrun{
 #' route("/", shiny::tags$div(shiny::tags$span("Hello world")))
 #'
-#' route("/main", shiny::tags$div(h1("Main page"), p("Lorem ipsum.")))
+#' route("main", shiny::tags$div(h1("Main page"), p("Lorem ipsum.")))
 #' }
 #' @export
 route <- function(path, ui, server = NA) {
+  if (!missing(server)) {
+    warning("`server` argument in `route` is deprecated.")
```

Comment:
Suggestion:

We inform the user that `route` is deprecated but the user doesn't know if they should use a different argument or just skip it. What do you think about being explicit about how the user should interpret deprecation here?

```suggestion
warning("`server` argument in `route` is deprecated. It does not need to be specified anymore.")
````

Reply:
Similarly to how you are explicit when deprecating `make_router`: https://github.com/Appsilon/shiny.router/blob/kuba.rhino-compatibility/R/router.R#L304-L307